File: desafios_de_codigo/python/salario.py
def calcular_imposto(salario):
    alicota = 0.00
    if (salario >= 0 and salario <= 1100):
        alicota = 0.05

    return alicota * salario

valor_salario = float(input())
valor_beneficios = float(input())

valor_imposto = calcular_imposto(valor_salario)

saida = valor_salario - valor_imposto + valor_beneficios
print(f"{saida:.2f}")

File: desafios_de_codigo/javaScript/salario.js
const valorSalario = parseFloat(gets());
const valorBeneficios = parseFloat(get());

const valorImposto = calcularImposto(valorSalario);

const saida = valorSalario - valorImposto + valorBeneficios;
print(saida.toFixed(2));

function calcularImposto(salario) {
    let alicota;
    if (salario >= 0 && salario <= 1100) {
        alicota = 0.05;
    } else if (salario >= 1100.01 && salario <= 2500.00) {
        //Atribui a licota de 5% mediante salario:
        alicota = 0.10;
    } else if (salario >= 2500.01){
        //Atribui a licota de 5% mediante salario:
        alicota = 0.15;
    }

    return alicota * salario;
}

